assets/src/edit-story/components/richText/karma/inlineSelection.karma.js

```diff
@@ -96,7 +96,9 @@ describe('CUJ: Creator can Add and Write Text: Select an individual word to edit
 
       // Set color using hex input
       await data.fixture.events.click(fontColor.hex, { clickCount: 3 });
-      await data.fixture.events.keyboard.type('FF00FF');
+      await data.fixture.events.keyboard.type('F60');
```

Comment:
There are still tests setting a 6 digit hex, or?

Reply:
Yes, in this inlineSelection file, it looks like there isn't a 6 digit hex test.  But there are both 3 digit and 6 digit hex assertions in the single and multi selection karma files.  I'll add in an extra assertion to this inline file so we have a 6 digit hex example too!